docs/source/api/apollo-server.mdx

```diff
@@ -618,6 +619,78 @@ This method takes no arguments. You should only call it after [`start()`](#start
 
 In some circumstances, Apollo Server calls `stop` automatically when the process receives a `SIGINT` or `SIGTERM` signal. See the [`stopOnTerminationSignals` constructor option](#stoponterminationsignals) for details.
 
+
+## `executeOperation`
+
+The async `executeOperation` method is used primarily for [testing GraphQL operations](../testing/testing/#testing-using-executeoperation) through Apollo Server's request pipeline _without_ sending an HTTP request.
+
+```js
+const result = await server.executeOperation({
+  query: 'query SayHelloWorld($name: String) { hello(name: $name) }',
+  variables: { name: 'world' },
+});
+```
+
+The `executeOperation` method takes two arguments:
+* The first argument is a configuration options object.
```

Comment:
I think I'd say more that the first option describes the operation, or something. It's roughly the same as the body of a POSTed GraphQL operation though, eg, it allows you to specify the operation as a pre-parsed AST (DocumentNode) as well as a stirng.